content/docs/user-guide/experiment-management/comparing-experiments.md

```diff
@@ -348,6 +348,20 @@ $ dvc exp show --csv | csvstat
 
 [csvkit]: https://csvkit.readthedocs.io/en/latest/
 
+## Show the experiments table as a Parallel Coordinates plot
+
+`dvc exp show` can also generate an interactive Parallel Coordinates plot, with
```

Comment:
Hmmm. How is a table anymore? 🙂

Reply:
Indeed, it's not a table anymore, but it does contain the same data as the table. 

So, it's a different representation of the same data. 

`--json` output is neither a table and it's named `Get experiments table in JSON`

Reply:
"Experiments table" refers to the output of `dvc exp show` in default format. When you supply `--json`, you get the content of this table in JSON format.

Here we can call "Show experiment results as a PCP" not to confuse.

Reply:
I'm ok with any title. Just wanted to point out that the content of the PCP plot is the same as the `exp show` table, just with a different representation. 

On the contrary, the `--json` output contains different content to the `exp show` table but it's treated similar to other output types (`--csv`).

The `--json` output is dumped way before we create and apply flags to the table and it's content is very different:

https://github.com/iterative/dvc/blob/main/dvc/command/experiments/show.py#L543-L573

Reply:
Is it a discussion on the title `Show the experiments table as a Parallel Coordinates plot` or something else? Title sounds fine to me personally (even though I still think that naming option `--html` can lead to confusion now and some potential renamings in the future).

Reply:
Originated with the title but I guess that it is going a little in circles and it's really about https://github.com/iterative/dvc/issues/7165

Reply:
Technically it may be the other way around, `dvc exp show` may be the beautified, filtered and tabulated version of `--json` output but from the user POV, `--json` contains a similar data. If you're fine with "experiments table", that also feels fine with me.

Reply:
> --json output is neither a table and it's named Get experiments table in JSON

Good point I guess we could separately update that title too... See https://github.com/iterative/dvc.org/pull/3137

I see the example title in question has been updated, thanks. Resolving!